configs/guided_diffusion/adm-u_8xb32_imagenet-512x512.py

```diff
@@ -0,0 +1,45 @@
+_base_ = [
+    '../_base_/datasets/imagenet_512.py',
```

Comment:
I think you can provide a draft readme here, which provides citation and credits for guided_diffusion, and some parts can be remained as 'coming soon'